main.py

```diff
@@ -57,6 +57,37 @@ def construct_fts_params_dict(event, pid, file_extension, dest_path, ctx):
     }
 
 
+def construct_mediahaven_external_metadata(event, pid):
+    ###############################
+    # DIRTY MEDIAHAVEN WORKAROUND #
+    ###############################
+    s3_object_key = get_from_event(event, "object_key")
+
+    root = etree.Element("MediaHAVEN_external_metadata")
+    etree.SubElement(root, "title").text = f"Essence: pid: {pid}"
+    etree.SubElement(
+        root, "description"
+    ).text = f"Main fragment for essence:\n- filename: {s3_object_key}\n- CP: VRT\n"
```

Comment:
```suggestion
    description = f"""Main fragment for essence:
    - filename: {s3_object_key}
    - CP: VRT
    """
    etree.SubElement(root, "description").text = description
```

Kan dit werken? Is dit properder?